types.d.ts

```diff
@@ -246,18 +246,18 @@ declare interface ArgumentConfig {
 }
 type ArrayBufferLike = ArrayBuffer | SharedArrayBuffer;
 type ArrayBufferView<TArrayBuffer extends ArrayBufferLike = ArrayBufferLike> =
-	| Uint8Array
-	| Uint8ClampedArray
-	| Uint16Array
-	| Uint32Array
-	| Int8Array
-	| Int16Array
-	| Int32Array
-	| BigUint64Array
-	| BigInt64Array
-	| Float32Array
-	| Float64Array
-	| DataView;
+	| Uint8Array<TArrayBuffer>
+	| Uint8ClampedArray<TArrayBuffer>
+	| Uint16Array<TArrayBuffer>
+	| Uint32Array<TArrayBuffer>
+	| Int8Array<TArrayBuffer>
+	| Int16Array<TArrayBuffer>
+	| Int32Array<TArrayBuffer>
+	| BigUint64Array<TArrayBuffer>
+	| BigInt64Array<TArrayBuffer>
+	| Float32Array<TArrayBuffer>
+	| Float64Array<TArrayBuffer>
+	| DataView<TArrayBuffer>;
```

Comment:
Please update your dependencies locally, looks like you have old `webpack/tooling` version